fastlane/Fastfile

```diff
@@ -188,6 +199,13 @@ platform :ios do
     app_store_connect
     begin
       case options[:scheme]
+      when 'DevOrange'
+        upload_to_testflight(
+          ipa: 'iosApp/build/iosApp.ipa',
+          distribute_external: true,
+          changelog: '',
+          groups: ['Dev Orange Users']
```

Comment:
question(non-blocking): Is there a reason we'd want to manage this group separately from our existing groups? Wondering if we only need one group with access to all non-prod apps for simplicity

Reply:
It doesn’t look like TestFlight groups carry over from one app to another.